DOCS/man/options.rst

```diff
@@ -4338,11 +4338,7 @@ The following video options are currently all specific to ``--vo=opengl`` and
     v-log
         Panasonic V-Log (VARICAM) curve
 
-    NOTE: When using HDR output formats, mpv will encode to the specified
-          curve but it will not set any HDMI flags or other signalling that
-          might be required for the target device to correctly display the
-          HDR signal. The user should independently guarantee this before
-          using these signal formats for display.
+    NOTE: When using HDR output formats, mpv will encode to the specified curve but it will not set any HDMI flags or other signalling that might be required for the target device to correctly display the HDR signal. The user should independently guarantee this before using these signal formats for display.
```

Comment:
This isn't readable in a text editor without wrapping.
Just remove the extra indentation.